app/serializers/deployment_serializer.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# Serializer for Deployment model
+class DeploymentSerializer < BaseSerializer
+  attributes :name, :created_at, :updated_at
+
+  belongs_to :app, links: {
+    related: lambda { |object|
+      "#{api_path}/teams/#{object.app.team_id}/apps/#{object.app.id}"
+    }
+  }
+
```

Comment:
Do we also want to include the argo related deployment info that we get from the API call to argo? e.g. [this show action](https://github.com/department-of-veterans-affairs/platform-console-api/blob/master/app/controllers/deployments_controller.rb#L18-L25)

Reply:
Yep I can add that, I missed that. Good catch

Reply:
Going to punt on this for now until we get the `keycloak_token` stuff sorted out.